turbopack/crates/turbopack-node/src/pool.rs

```diff
@@ -849,6 +863,26 @@ impl NodeJsPool {
             allow_process_reuse: true,
         })
     }
+
+    pub fn scale_down() {
+        let pools = ACTIVE_POOLS.lock().clone();
+        for pool in pools {
+            let mut pool = pool.lock();
+            let best = pool.pop().unwrap();
```

Comment:
There might be no idle process, then this would throw?

Reply:
It will be removed from `ACTIVE_POOLS` when it's empty. So it will only have non-empty lists.